modules/contact-form/grunion-contact-form.php

```diff
@@ -2423,6 +2423,19 @@ function process_submission() {
 
 		update_post_meta( $post_id, '_feedback_extra_fields', $this->addslashes_deep( $extra_values ) );
 
+		$form_data = array();
+		foreach ( $this->fields as $key => $field ) {
+			$form_data[ $key ] = array(
+				'key'     => $key,
+				'value'   => $field->value,
+				'type'    => $field->attributes['type'],
+				'label'   => $field->attributes['label'],
+				'options' => isset( $field->attributes['options'] ) ? $field->attributes['options'] : '',
+			);
+		}
+		// capture all feedback keys, values, and types so we have it with delimiters in the future
```

Comment:
Not blocking, but I wonder if it may be useful to state what that data will be used for here. We have quite a bit of post meta data, it may be useful to know what this is used for.